Lombiq.ContentEditors/Lombiq.ContentEditors.csproj

```diff
@@ -19,8 +21,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Include="Assets\Scripts\async-editor.js">
+      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+    </None>
     <None Include="License.md" Pack="true" PackagePath="" />
     <None Include="NuGetIcon.png" Pack="true" PackagePath="" />
+    <None Remove="Assets\Scripts\async-editor\async-editor.js" />

```

Comment:
These changes look undesired. Did you move the _async-editor.js_ file out of the _async-editor_ folder into _Assets\Scripts_?

Then you should be able to revert these changes here.

Reply:
You are correct, Rider sometimes puts stuff to the csproj after I add or remove files. Thank you for pointing out!